providers/aws/resources.ts

```diff
@@ -4340,15 +4340,27 @@ export function fieldsFromLambdaPermissionParams(params: LambdaPermissionParams)
 
 export interface CloudwatchEventRuleParams {
   name?: string;
+  name_prefix?: string;
   schedule_expression?: string;
+  event_bus_name?: string;
```

Comment:
`event_bus_name?: string;` not available on AWS provider 2.x

Reply:
Sigh. Removed